docs/installation-configuration.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Mist Provider Installation & Configuration
+meta_desc: Provides an overview on how to configure the Pulumi Mist Provider.
+layout: package
+---
+
+## Installation
+
+The mist provider is available as a package in all Pulumi languages:
+
+* JavaScript/TypeScript: [`@pulumi/mist`](https://www.npmjs.com/package/@pulumi/mist)
+* Python: [`pulumi-mist`](https://pypi.org/project/pulumi-mist/)
+* Go: [`github.com/pulumi/pulumi-junipermist/sdk/go/junipermist`](https://github.com/pulumi/pulumi-mist)
+* .NET: [`Pulumi.Mist`](https://www.nuget.org/packages/Pulumi.Mist)
+* Java: [`com.pulumi/mist`](https://central.sonatype.com/artifact/com.pulumi/mist)
+
+
+The Mist Provider allows Pulumi to manage Juniper Mist Organizations.
+
+It is mainly focusing on day 0 and day 1 operations (provisionning and delpyment) but will be completed over time.
+
+Use the navigation tree to the left to read about the available resources and data sources.
+
+It is possible to use API Token or Username/Password authentication (without 2FA), but only one method should be configured.
+
+This version is supporting the following Mist Clouds:
+* Global 01 (api.mist.com)
+* Global 02 (api.gc1.mist.com)
+* Global 03 (api.ac2.mist.com)
+* Global 04 (api.gc2.mist.com)
+* EMEA 01 (api.eu.mist.com)
+* EMEA 02 (api.gc3.mist.com)
+* EMEA 03 (api.ac6.mist.com)
+* APAC 01 (api.ac5.mist.com)
+
+## Configuration
+
+### Provider configuration stanza
+
+```pulumi
+pulumi {
+  required_providers {
+    mist = {
+      source = "registry.pulumi.io/juniper/mist"
+    }
+  }
+}
+
+provider "mist" {
+  host     = "api.mist.com"
+  apitoken = "xxxxxxxxxxxxxxxxxxxxxxxxxxx"
+}
+```
```

Comment:
@mjeffryes This code isn't turned on for a reason. Of special significance, we are missing https://github.com/pulumi/pulumi-terraform-bridge/pull/2214.

Reply:
Ok, that makes perfect sense, but the real question is: How did these docs/* pages get here for other providers and why didn't that happen for this one? 

I checked other providers bridged recently (eg. dbtcloud) and these files we generated.

Reply:
I have no idea. I thought that they were all hand authored. Until recently, the bridge didn't generate them.

Is it possible that you generated them the same way for dbtcloud as this provider, but the bridge just did something different. We are not maintaining stability guarantees on this feature yet.

Reply:
That's weird.

Eg. I made the initial commit for dbtcloud as well: [Initial commit · pulumi/pulumi-dbtcloud@9e22c8b](https://github.com/pulumi/pulumi-dbtcloud/commit/9e22c8be4d8a1cbc1c158aa278a77c19ae23e01e#diff-6ea303a35b3d9378eea0299496163e7c837b60501391c587a50dbc213038fce7) but definitely didn't hand author the files in docs/ for that provider.